package exception;

public class Main {

	public static void main(String[] args) {
		
		
		
		
	System.out.println("V채lkommen till hela detta programmet! ");
		
		
		
		
		
		try {
			
			
		String[]name = {"Anna " , "Johan"};
		
		System.out.println(name[2]);
		
		
		
	       }  catch (ArrayIndexOutOfBoundsException e){
	    	   
	    	   System.out.println("Detta 채r Array Exception och den 채r: " + e);
	        	 
	        	 
	        	 
	        
		
		
			 } catch  (Exception e) {
				 System.out.println("Opps n책t gick fel");
			
		
		}
		
	

		System.out.println("hoppas du har haft det bra");
		
		
	}

}
